### DISCUSSION TITLE
Automatically add voters to whitelist

### DISCUSSION THREAD

**Discussion by valentinfernandez1** [snapshot-labs/snapshot#3212] [Q&A]
Hi! I'm new to snapshot. I want to integrate snapshot to my app by adding each new user of the platform to vote on proposals. Is there a way to add voters to the whitelist automatically using an API call or in some way? 

Whenever a new user signs up in my platform with their wallet address I want to add them to the list.

**Comment by metrox-eth** [comment]
This would require to create a custom strategy (or maybe using the API strategy ?) @ChaituVR will better answer this question, I let you in good hands.

**Comment by ChaituVR** [comment]
Hey @valentinfernandez1 Yes you can use `api` strategy for this, return voting power for the addresses, 
If your API doesn't know the voting power, you can try using [validation strategy](https://snapshot.org/#/strategy/validation) to use api strategy to validate/whitelist and other strategies to calculate voting power

**Reply by valentinfernandez1** [reply]
Great, the idea would be to have 1 vote per user so each vote has the same power.

So i'll create an endpoint that returns the addreses that are elegible to vote on snapshot
